config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml

```diff
@@ -3,15 +3,11 @@ kind: OpenStackDataPlaneNodeSet
 metadata:
   name: openstack-edpm-ipam
 spec:
-  ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
   env:
     - name: ANSIBLE_FORCE_COLOR
       value: "True"
     - name: ANSIBLE_ENABLE_TASK_DEBUGGER
       value: "True"
-  preProvisioned: true
```

Comment:
What's the reason for dropping this? We should decide on what we want the samples to represent, all fields with default values, or the minimal set of fields. For instance, why drop preProvisioned, but keep env?